lib/package.gi

```diff
@@ -213,7 +213,7 @@ end );
 #F  AddPackageInfo( files )
 ##
 BindGlobal( "AddPackageInfos", function( files, pkgdir, ignore )
-    local file, record, pkgname, version;
+    local file, record, pkgname, version, date, dd, mm;
```

Comment:
So, we call `ValidatePackageInfo` from `LoadPackage` but not from `AddPackageInfos`. I wonder if we should do that in `AddPackageInfos` too...

Reply:
Perhaps, but that goes beyond the purpose of this PR, doesn't it?